tests/playbooks/kubevirt_facts_template.yml

```diff
@@ -0,0 +1,21 @@
+---
+- name: Gather facts for Templates
+  hosts: localhost
+  connection: local
+  vars:
+    ansible_python_interpreter: "/usr/bin/env python"
```

Comment:
What is the reason to use this? I think we should keep the playbook as simple as possible.